.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,37 @@
+
+----
+
+### Pull Request Checklist
+
+Please check all boxes (including N/A items)
+
+#### Testing
+
+- [ ] Unit test and/or integration test added
+- [ ] __Toolkit change?:__ [integration
+tests](https://github.com/awslabs/aws-cdk/blob/master/packages/aws-cdk/integ-tests/test.sh)
+manually executed (paste output to the PR description)
+- [ ] __Init template change?:__ coordinated update of integration tests
+(currently maintained a private repo).
+
+#### Documentation
+
+- [ ] __README__: README and/or documentation topic updated
+- [ ] __jsdocs__: All public APIs documented
+
+### Title and description
+
+- [ ] __Change type__: Title is prefixed with change type:
+ * `fix(module): <title>` bug fix (_patch_)
+ * `feat(module): <title>` feature/capability (_minor_)
+ * `chore(module): <title> ` won't appear in changelog
+ * `build(module): <title>` won't appear in changelog
+- [ ] __Title format__: Title uses lower case and doesn't end with a period
+- [ ] __Breaking change?__: Description ends with: `BREAKING CHANGE: <one
```

Comment:
Multiple paragraphs are included though, right?

Reply:
They are but the CHANGELOG looks really bad in that case, so I rather limit to a single paragraph. Added clarification